FlowersStore/Scenes/FlowersList/TestPeriphery.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  TestPeriphery.swift
+//  FlowersStore
+//
+//  Created by Leonir Deolindo on 02/05/22.
+//
+
+import Foundation
+import UIKit
+
+protocol Test {
+    func something()
+}
+
+struct SeLiga {
+    let name: String
+}
+
+class OlhaLesk {
```

Comment:
<!--
  0 failure: 
  1 warning:  Class 'OlhaLesk' ...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStore/Scenes/FlowersList/TestPeriphery.swift;
  Line: 19;
-->

- :warning: Class 'OlhaLesk' is unused